.gitignore

```diff
@@ -2,3 +2,6 @@
 **/*.rs.bk
 Cargo.lock
 .vscode
+.idea
+action-circuit-layout.png
+proptest-regressions/*.txt
```

Comment:
proptest regressions should always be persisted, because we want those regression cases tested by future proptest runs.

Reply:
Thanks, done as part of https://github.com/QED-it/orchard/pull/35